torch/_inductor/config.py

```diff
@@ -13,6 +13,20 @@
 # Whether to enable printing the source code for each future
 verbose_progress = False
 
+# AOT compile model into .so file
+aot_codegen = False
+
+# Name for generated .h and .so files
+aot_codegen_output_prefix = "aot_inductor_output"
+
+
+def set_aot_codegen():
+    global aot_codegen
+    aot_codegen = True
+    global debug
+    debug = True
+
+
```

Comment:
I don't like the UX of using `config` to change behavior.

I'd suggest adding a new function like `aot_compile_fx(graph, example_inputs)` instead.  You can put mode flags on V.graph rather than in the config.